app/models/subscription/GroupMembers.scala

```diff
@@ -17,28 +17,30 @@
 package models.subscription
 
 import models.subscription.subscriptionDisplay.SubscriptionDisplayResponse
+import play.api.libs.json.{Format, Json}
 
-case class GroupMembers(membersNames: Seq[String]) {
-  def contains: String => Boolean = membersNames.contains
-
-  def map[A](function: (String, Int) => A): Seq[A] =
-    membersNames.zipWithIndex.map(tuple => function(tuple._1, tuple._2))
+case class Member(organisationName: String, crn: String)
+object Member {
+  implicit val formats: Format[Member] = Json.format[Member]
```

Comment:
we need a way to share code between repos

Reply:
agreed! this types of changes are annoying